### ISSUE TITLE
js for data interaction website

### ISSUE THREAD

**Issue by jessegmeyerlab** [czbiohub/tabula-muris#249]
For the website https://tabula-muris.ds.czbiohub.org/ we are wondering how the javascript was implemented for the interactive data update. Did you pre-generate all the images and grab them as they are requested or is the server computing the new image on the fly and sending it?

**Comment by aopisco** [comment]
@neevor is the best person to comment regarding the details, but in a nutshell the images were all pre-computed and then just grabbed

**Comment by neevor** [comment]
That's correct. They are all pre-generated and then the JS just updates the
img src value to the selected values. It's a very unsophisticated approach
:).

**Comment by jessegmeyerlab** [comment]
Thanks. When you have static data this seems like the best way. We were
thinking about how to do this on a repository scale when you can't afford
to store all the images. Even sending all the data to the browser and using
d3 for vis seems infeasible when you get into hundreds of MB per dataset.
Probably need some back and forth with server. Any ideas?

**Comment by aopisco** [comment]
For Tabula Sapiens @kyleawayan built these visualizations https://tabula-sapiens-portal.ds.czbiohub.org/summaryvisualizations but what we are also doing is self-hosting cellxgene e.g. https://tabula-sapiens-portal.ds.czbiohub.org/organs
